### PR TITLE
Update en_US.json once more

### PR DESCRIPTION
**Pull request by KingPanda360**

### CODE REVIEW

File: Resources/Lang/en_US.json [hunk 1]
```diff
@@ -688,7 +688,7 @@
...
   "AnonymousInfoLong": "(Impostors):\nAs the Anonymous, you can Shapeshift to force your target to report whoever you killed this round.\nIf you killed nobody that round, the target will report their own dead body as if they had died.\nNote: This does not work on Lazy nor Lazy Guy, and this ability will work regardless of whether the body can normally be reported.",
   "MinerInfoLong": "(Impostors):\nAs the Miner, you can shapeshift to teleport back to the last vent you were in.",
-  "KillingMachineInfoLong": "(Impostors):\nAs the Killing Machine, you have a very short kill cooldown with tiny vision. However, you cannot vent, sabotage, report, and call emergency meetings.\n\nNote: You will bypass any shields, killing bait and beartrap won't take any effect",
+  "KillingMachineInfoLong": "(Impostors):\nAs the Killing Machine, you have a very short kill cooldown with tiny vision. However, you cannot vent, sabotage, report, or call emergency meetings.\n\nNote: You will bypass any shields, killing bait and beartrap won't take any effect",
```
Comment (Ultradragon005) [inline Resources/Lang/en_US.json:691]: it's nor

Comment (KingPanda360) [reply Resources/Lang/en_US.json:691]: "nor" would be incorrect word choice. "or" a better conjunction to use here.

Comment (Ultradragon005) [reply Resources/Lang/en_US.json:691]: image
idk what to tell u

Comment (Ultradragon005) [reply Resources/Lang/en_US.json:691]: image
it even has it in the definiton

Comment (KingPanda360) [reply Resources/Lang/en_US.json:691]: either one can be used here. but if you want to use "nor" we can.

**Pull request merged**